### ISSUE TITLE
insert frontend commit version somewhere in the UI, in  the footer?

### ISSUE THREAD

**Issue by volodymyrss** [oda-hub/mmoda-frontend-module#255]

**Comment by burnout87** [comment]
are we talking about the version, as discussed here ? https://github.com/oda-hub/frontend-tab-generator/issues/35

**Comment by volodymyrss** [comment]
> are we talking about the version, as discussed here ? [oda-hub/frontend-tab-generator#35](https://github.com/oda-hub/frontend-tab-generator/issues/35)

no, here I am talking about frontend version, not instrument version. Frontend version consists of version of the involved component.

**Comment by burnout87** [comment]
What about 

![image](https://github.com/oda-hub/mmoda-frontend-module/assets/3757833/6d21f3f0-bd3d-4348-9e48-1e5a73113f4f)

**Comment by volodymyrss** [comment]
looks good!

**Comment by burnout87** [comment]
how are you planning to update the version when there is a new deployment ?

**Comment by burnout87** [comment]
this issue was about the frontend version, for the version of the specific instrument, it was [here](https://github.com/oda-hub/frontend-tab-generator/pull/36/files), it can  be added as an arg from the tab_generator CLI:

![image](https://github.com/user-attachments/assets/b5a74606-e6c2-45a2-a1f7-333c66c9051b)

The args are:

* `instrument_version`
* `instrument_version_link`
